conf/messages

```diff
@@ -747,3 +747,32 @@ calc.resident.shares.currentIncome.helpText = Include your salary before tax, an
 ## Other Disposals messages
 calc.resident.shares.otherDisposals.bullet.one = UK residential properties
 calc.resident.shares.otherDisposals.bullet.three = other shares
+
+## Final Summary messages
+calc.summary.cgtToPay = Capital Gains Tax to pay for the {0} tax year
+calc.summary.calculationDetails.heading = How we''ve worked this out
+calc.summary.calculationDetails.yourTotalGain = Your total gain
+calc.summary.calculationDetails.disposalValue = Value when you sold the shares
+calc.summary.calculationDetails.marketValue = Value when you gave away the shares
+calc.summary.calculationDetails.acquisitionValue = Minus the value of the shares when you acquired them
+calc.summary.calculationDetails.acquisitionValueBeforeLegislation = Minus the value of the shares at 31 March 1982
+calc.summary.calculationDetails.totalCosts = Minus all costs
+calc.summary.calculationDetails.totalGain = Total gain
+calc.summary.calculationDetails.yourDeductions = Your deductions
+calc.summary.calculationDetails.aeaUsed = Capital Gains Tax Annual Exempt Amount used
+calc.summary.calculationDetails.lossesUsed = Loss used from previous tax years
+calc.summary.calculationDetails.totalDeductions = Total deductions
+calc.summary.calculationDetails.yourTaxableGain = Your taxable gain
+calc.summary.calculationDetails.minusDeductions = Minus deductions
+calc.summary.calculationDetails.taxableGain = Taxable gain
+calc.summary.calculationDetails.yourTaxRate = Your tax rate
+calc.summary.calculationDetails.taxRate = £{0} taxable gain multiplied by {1}% tax rate
+calc.summary.calculationDetails.taxToPay = Tax to pay
+calc.summary.remainingDeductions.heading = Your remaining deductions
+calc.summary.remainingDeductions.aeaLeft = Annual Exempt Amount left for the {0} tax year
+calc.summary.remainingDeductions.broughtForwardLossesRemaining = Losses to carry forward from previous tax years
+calc.summary.remainingDeductions.lossesToCarryForward = Losses you can carry forward from this calculation
+calc.summary.whatToDoNext.heading = What to do next
```

Comment:
This is a duplicate key, could you either rename this one or delete the previous one (but update where it is used already)